src/data_info/services/manifest.clj

```diff
@@ -68,7 +67,7 @@
 (defn do-manifest-uuid
   [user data-id]
   (irods/with-jargon-exceptions [cm]
-    (let [file (uuids/path-stat-for-uuid cm user data-id)]
+    (let [file (stat/uuid-stat cm user data-id :filter-include [:path :content-type :infoType])]
```

Comment:
This filter is chosen by looking around the rest of this file. The `manifest` function uses the path, and then `manifest-map` does a `select-keys` down to `:infoType` and `:content-type`.